### DISCUSSION TITLE
【建议】举报帖子的使用方法

### DISCUSSION THREAD

**Discussion by Hongxuli2006** [share121/inter-knot#1776] [其他]
![生气的三月七](https://github.com/user-attachments/assets/5100c8b7-5c1a-4aa8-825e-08af5207e2b1)

大家别把整活的帖子比如整活儿【广告】帖子，18-帖子给举报了啊。挺有意思的那些。

**Comment by AnbulesPeo** [comment]
还是得等到具体的社区规则完善之后再说吧，要不然有的帖子很难界定如何举办啊

**Reply by AnbulesPeo** [reply]
我的意思是说“是否应该举办”
